get_together/templates/get_together/new_event/pick_team.html

```diff
@@ -55,10 +55,9 @@ <h3>{% trans "You don't have any teams that can host this event, would you like
         <div class="col-12">
             <center>
                 <a class="btn btn-success" href="{% url 'create-team' %}?event={{event.id}}">{% trans "Create a new Team" %}</a>
-                <a class="btn btn-secondary" href="{{event.get_absolute_url}}">{% trans "Host this event as myself" %}</a>
+                <a class="btn btn-secondary" href="{{event.get_absolute_url}}">{% trans theme.lables.host_as_myself %}</a>
```

Comment:
Typo here; It doens't throw any error, but the button text doesn't show up. 
```suggestion
                <a class="btn btn-secondary" href="{{event.get_absolute_url}}">{% trans theme.text_labels.host_as_myself %}</a>
```